packages/esm-appointments-app/src/appointment-forms/create-appointment-form.component.tsx

```diff
@@ -0,0 +1,383 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSWRConfig } from 'swr';
+import dayjs from 'dayjs';
+import {
+  Button,
+  DatePickerInput,
+  DatePicker,
+  SearchSkeleton,
+  Select,
+  SelectItem,
+  TextArea,
+  TimePickerSelect,
+  TimePicker,
+  ContentSwitcher,
+  Switch,
+  FormGroup,
+  RadioButtonGroup,
+  RadioButton,
+  Toggle,
+} from 'carbon-components-react';
+import {
+  useLocations,
+  useSession,
+  showToast,
+  showNotification,
+  useLayoutType,
+  ExtensionSlot,
+  useConfig,
+} from '@openmrs/esm-framework';
+import {
+  appointmentsSearchUrl,
+  createAppointment,
+  useServices,
+  useProviders,
+  fetchAppointments,
+} from './appointment-forms.resource';
+import { AppointmentPayload } from '../types';
+import { convertTime12to24, amPm } from '../helpers/time.helpers';
+import { ConfigObject } from '../config-schema';
+import { mockFrequency } from '../../../../__mocks__/appointments.mock';
+import { closeOverlay } from '../hooks/useOverlay';
+
+import styles from './create-appointment-form.scss';
+
+interface AppointmentFormProps {
+  patientUuid: string;
+  patient: fhir.Patient;
+}
+
+const CreateAppointmentsForm: React.FC<AppointmentFormProps> = ({ patientUuid, patient }) => {
+  const { appointmentKinds } = useConfig() as ConfigObject;
+  const { daysOfTheWeek } = useConfig() as ConfigObject;
+  const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+  const { mutate } = useSWRConfig();
+  const locations = useLocations();
+  const session = useSession();
+  const [appointmentNote, setAppointmentNote] = useState('');
+  const [selectedProvider, setSelectedProvider] = useState('');
+  const [selectedService, setSelectedService] = useState('');
+  const [startDate, setStartDate] = useState(new Date());
+  const [startTime, setStartTime] = useState(dayjs(new Date()).format('hh:mm'));
```

Comment:
Is it possible to change this to be just one `Date` object? UTC is much more reliable than recording date and time separately, especially looking to future proof against timezones

Reply:
Yah, I was thinking of that too, I will add it to follow up PR